### ISSUE TITLE
Setting spawn rate to 0 wont stop mob from spawning

### ISSUE THREAD

**Issue by IvanNSBS** [Silentine/GrimoireOfGaia#51]
Pretty much what the titlte says 
GoG tested version : 1.10.2 - 1.5.4 to 1.5.6

**Comment by ChaoticLogic** [comment]
Can confirm. Altering the spawn values ranging between 1 and 100 doesn't seem to have effect. I tested this in a snow biome where selkies prevented me from starting the game by overwhelming force/numbers. Really love the mobs, but would like them to be much more rare.

**Comment by MokahTGS** [comment]
Yes, the spawn rate configs is the main reason we haven't implemented this in our modpack.

**Comment by P3rf3ctXZer0** [comment]
I can confirm this is 100% an issue.

**Comment by Mrbysco** [comment]
Have you restarted the game after editing the settings?

**Comment by P3rf3ctXZer0** [comment]
Of course. Seems to only break when on a server.

**Comment by Silentine** [comment]
I can confirm that this works in singleplayer. I cannot confirm if it works in multiplayer though.
Perhaps restarting the server could do the trick after making changes to the config.

**Comment by IamKayrox** [comment]
I recommend using "Custom Mob Spawner"

**Comment by junkerprime** [comment]
Can confirm, in a server/client environment, am getting way too many mobs, configs have no effect.  Neither does placing talismans
